src/traces/candlestick/index.js

```diff
@@ -8,13 +8,11 @@
 
 'use strict';
 
-var Registry = require('../../registry');
-
 module.exports = {
     moduleType: 'trace',
     name: 'candlestick',
     basePlotModule: require('../../plots/cartesian'),
-    categories: ['cartesian', 'showLegend', 'candlestick'],
+    categories: ['cartesian', 'showLegend', 'candlestick', 'boxLayout'],
```

Comment:
So users making graphs with candlestick traces are allowed to set `layout.boxgap` and friends?

Reply:
yes... I included `boxgroupgap` in the mock in https://github.com/plotly/plotly.js/pull/2561/commits/0b7541e34d63ce74cddb5d243a652b2df0430b70 to 🔒 it